tornjak-frontend/src/components/dashboard/entries-dashboard-table.js

```diff
@@ -42,15 +43,15 @@ class EntriesDashBoardTable extends React.Component {
     })
 
     // For details page filtering data
-    if (selectedDataKey !== undefined) {
-      for (let i = 0; i < entriesList.length; i++) {
-        if ((entriesList[i].clusterName === selectedDataKey) || (entriesList[i].parentId === selectedDataKey)) {
-          filterByValue.push(entriesList[i]);
-        }
+    if (filterByCluster === undefined && filterByAgentId === undefined) {
+      return entriesList;
+    }
+    for (let i = 0; i < entriesList.length; i++) {
+      if ((filterByCluster !== undefined && entriesList[i].clusterName === filterByCluster) || (filterByAgentId !== undefined && entriesList[i].parentId === filterByAgentId)) {
```

Comment:
nit: make this two lines for readability

Reply:
done!